test/shared-class.test.js

```diff
@@ -29,11 +29,6 @@ describe('SharedClass', function() {
       expect(sc.http.path).to.equal('/some');
     });
 
-    it('fills http.path using a normalized path', function() {
-      var sc = new SharedClass('SomeClass', SomeClass, {normalizeHttpPath: true});
```

Comment:
We need to preserve this behaviour for backwards compatibility (existing applications should keep working after upgrading to the new version).

With two places where to configure `normalizeHttpPath`, we need to decide which one takes priority - model-level or app-level configuration?

@raymondfeng @ritch @superkhau do you happen to remember any example of a config option that can be set both on app-level and model-level? I think we should make `normalizeHttpPath` consistent with the rest of the existing options.